test/html/converter_test.rb

```diff
@@ -328,6 +328,62 @@ def setup
     @converter.instance_variable_set(:@numbering, Sablon::Environment.new(nil).numbering)
   end
 
+  def test_node_properties_converison
```

Comment:
Instead of this single huge method, Can we create a new TestCase class in this file and split each example separated by a comment into a method?

```
class NodePropertiesTest < Sablon::TestCase
  def test_empty_properties
  end

  def test_simple_properties
  end

  ...
end
```